app/_locales/it/messages.json

```diff
@@ -618,15 +650,25 @@
   "externalExtension": {
     "message": "Estensione Esterna"
   },
+  "extraApprovalGas": {
+    "message": "+$1 gas approvazione",
+    "description": "Expresses an additional gas amount the user will have to pay, on top of some other displayed amount. $1 is a decimal amount of gas"
+  },
   "failed": {
     "message": "Fallita"
   },
+  "failedToFetchChainId": {
+    "message": "Non è stato possibile ottenere il blockchain ID. L'URL RPC è corretto?"
+  },
   "failureMessage": {
     "message": "Qualcosa è andato storto e non è possibile completare l'azione"
   },
   "fast": {
     "message": "Veloce"
   },
+  "fastest": {
+    "message": "Più veloce"
```

Comment:
Shouldn't this be "Il più veloce"

Reply:
I think is fine to leave it like this, because as i understand, this message in shown only on the transaction speed-up so is fine to don't have the subject as it's implicit.

![immagine](https://user-images.githubusercontent.com/28830727/109391226-565b5680-7916-11eb-8b01-068a79545608.png)

Although is not shown anymore so maybe is used somewhere else?

Otherwise it could be "Velocissima" which is a less "serious" term.

Reply:
@jacobcantele This appears to be the only blocker -- any thoughts?

Reply:
@matteopey is right here, I take my feedback back :)